guides/common/modules/proc_creating-a-report-template.adoc

```diff
@@ -39,3 +31,15 @@ If this field remains undefined, the users receive a free-text field in which th
 . Click the *Location* tab and add the locations where you want to use the template.
 . Click the *Organizations* tab and add the organizations where you want to use the template.
 . Click *Submit* to save your changes.
+
+.Additional resources
+
+* For more information about safe mode, see xref:Report_Template_Safe_Mode_{context}[].
+
+* For more information about writing templates, see xref:Template_Writing_Reference_{context}[].
+
+* For more information about macros you can use in report templates, see xref:Template_Macros_{context}[].
+
```

Comment:
This should render the same format without empty lines.

Reply:
Thanks, fixed.